pkg/table/printer.go

```diff
@@ -0,0 +1,93 @@
+package table
```

Comment:
Consider replacing this hand-built module with `https://github.com/olekukonko/tablewriter`:

https://github.com/olekukonko/tablewriter